### PR TITLE
Miscellaneous updates and cleanup 

### PR DESCRIPTION
**Pull request by ssmusoke**
1. Enable the coreapps.relationships app which is disabled by default (required to display linking of mother-baby pairs) 

2. Fix links to the ART summary page and cleanup validation rule messages 

3. Correct SQL statement for the LinkExposedInfantToMotherTask which had an error so the task was not running along with a unit test 

4. Add require statements to show the patient dashboard widgets when specified criteria is met 
- EID only shown for children below 10 years
- MCH only shown for women 

5. Update the coreapps version which has the require statements feature

### CODE REVIEW

**Comment by slubwama** [comment]
@ssmusoke you forgot to include fingerprint in UgandaEMR

**Comment by ssmusoke** [comment]
@slubwama I did not forget - I was not made aware that it was to be included. Is it in BinTray already?

**Comment by slubwama** [comment]
@ssmusoke  its one of the things Vincent asked about. did you read his email clearly??

**Comment by ssmusoke** [comment]
You have been working on that so I figured you would be adding that onto
the package

File: api/src/main/java/org/openmrs/module/aijar/AijarActivator.java [hunk 1]
```diff
@@ -104,6 +104,9 @@ public void started() {
             appFrameworkService.disableApp("coreapps.visitByEncounterType");
             appFrameworkService.disableApp("coreapps.dataIntegrityViolations");
             
+            // enable apps
+            appFrameworkService.disableApp("coreapps.relationships");
```
Comment (slubwama) [inline api/src/main/java/org/openmrs/module/aijar/AijarActivator.java]: @ssmusoke The comment and the function don't match. comment is enable, the method call is to disable.

**Comment by slubwama** [comment]
@ssmusoke these are the errors I get when on this PR.https://pastebin.com/vrp8Ukgj

**Comment by slubwama** [comment]
@ssmusoke you didnt solve the issue of the errors that I sent you in bintray. thou the app is enabled according to the code you sent

**Pull request merged**